crates/proto/src/xfer/dnssec_dns_handle/mod.rs

```diff
@@ -790,23 +806,52 @@ where
     //  TODO: strip RRSIGS to accepted algorithms and make algorithms configurable.
     let verifications = rrsigs
         .iter()
-        .map(|rrsig| {
+        .enumerate()
+        .filter_map(|(i,rrsig)| {
```

Comment:
Nit: I'm surprised rustfmt doesn't trigger on this, but I'd expect a space after the comma here.